### PR TITLE
[TOAZ-334] Remove Azure preview user detection and form

### PR DESCRIPTION
**Pull request by rtitle**
## Summary of changes:
Remove allow-list-based check gating MSFT login, and Azure billing project creation.

### What
- Removed check on `alpha-azure-feature-group` for creating/listing Azure billing projects
- Removed form in `AzurePreview.ts`, but kept the splash page

### Why
- We would like to deploy this on 9/19 to support Azure Public Preview

### Testing strategy
- [ ]

### CODE REVIEW

**Review by aherbst-broad — APPROVED** [review]

File: src/pages/AzurePreview.js [hunk 1]
```diff
@@ -1,18 +1,11 @@
-import { Fragment, useCallback, useEffect, useState } from 'react';
```
Comment (rtitle) [inline src/pages/AzurePreview.js:1]: This page now just serves the splash page, but not the form. Once we get to GA in Nov we can remove this file entirely.

image

File: config/staging.json [hunk 2]
```diff
@@ -26,7 +26,6 @@
     "appId": "saturnnonprod-O9lUP",
     "apiKey": "AQEBBAEkx4iE2KxNyI7Wx08EwU1ycTM7E4FMSmaibbMUQxNU6uQvuAJt7fyABAtFYSYfgEE"
   },
-  "azurePreviewGroup": true,
```
Comment (cahrens) [inline config/staging.json:29]: For completeness, also remove from https://github.com/broadinstitute/terra-helmfile/blob/51adddd811b00a5c7d24c0c79ebb59ceb0562979/values/app/terraui/bee.yaml.gotmpl#L29.

Comment (rtitle) [reply config/staging.json:29]: Thanks! Forgot about that one

Comment (rtitle) [reply config/staging.json:29]: https://github.com/broadinstitute/terra-helmfile/pull/4602

**Review by cahrens — APPROVED** [review]
I don't know how to select an account on the sandbox that will show me the splash screen, but the code diff looks good.

**Pull request merged**